app/src/sharedTest/java/org/oppia/android/app/home/HomeActivityTest.kt

```diff
@@ -250,9 +268,141 @@ class HomeActivityTest {
     }
   }
 
+  @Test
+  fun testHomeActivity_recyclerViewIndex1_displaysStoriesForYouText() {
+    storyProgressTestHelper.markRecentlyPlayedForFractionsStory0Exploration0(
+      profileId = ProfileId.newBuilder().setInternalId(internalProfileId).build(),
+      timestampOlderThanAWeek = false
+    )
+    launch<HomeActivity>(createHomeActivityIntent(internalProfileId)).use {
+      testCoroutineDispatchers.runCurrent()
+      onView(withId(R.id.home_recycler_view)).perform(
```

Comment:
use helper function `scrollToPosition` present at the bottom of this file.

Reply:
Done.